control-plane/p4RuntimeSerializer.cpp

```diff
@@ -245,7 +251,7 @@ static int64_t getTableSize(const IR::P4Table* table) {
 
     auto sizeProperty = table->properties->getProperty("size");
     if (sizeProperty == nullptr) {
-        return defaultTableSize;  
+        return defaultTableSize;
```

Comment:
It's almost like whoever wrote this should turn on the setting in their editor to show whitespace at the end of lines.